medusa/providers/torrent/json/t411.py

```diff
@@ -92,20 +188,38 @@ def search(self, search_strings, age=0, ep_obj=None):
 
             for search_string in search_strings[mode]:
                 if mode != 'RSS':
-                    logger.log('Search string: {search}'.format
-                               (search=search_string), logger.DEBUG)
+                    season = episode = None
+                    if isinstance(search_string, tuple):
```

Comment:
For example:
```python
>>> import collections
>>> isinstance(tuple(), collections.Sequence)
True
>>> isinstance(list(), collections.Sequence)
True
>>> isinstance(set(), collections.Sequence)
False
>>> isinstance(collections.namedtuple('Test', 'a b c')(1, 2, 3), collections.Sequence)
True
>>> isinstance('', collections.Sequence)
True
```
True for everything except `set` since a set doesn't preserve order and so wouldn't work for this situation anyway.